packages/melos/bin/melos.dart

```diff
@@ -40,7 +40,13 @@ Future<void> main(List<String> arguments) async {
     return;
   }
   try {
-    final config = await MelosWorkspaceConfig.fromDirectory(Directory.current);
+    var allowOutside = false;
+    if (arguments.contains('--help') || arguments.contains('-h')) {
+      allowOutside = true;
+    }
```

Comment:
This can just be:

```dart
final hasHelpOption = arguments.contains('--help') || arguments.contains('-h');
```

Reply:
The reason i put it this way was so if in future we want more flags to be available outside the melos workspaces, we can update this condition here.

Reply:
How about this, then:
```dart
final hasHelpOption = arguments.contains('--help') || arguments.contains('-h');
final useEmptyConfig = hasHelpOption;
```
We avoid a mutable variable, it's shorter. If we later want to run with an empty config in other situations, we can add `|| <condition>` behind `hasHelpOption` and have an easily readable list of conditions when we need to use an empty config.

Reply:
Makes sense, may be as well just move it out to a function, 
```
bool shouldUseEmptyConfig(List<String> arguments)
```
And move all this there and use that

Reply:
That sounds good.

Reply:
Cool, will make the changes tomorrow, busy today

Reply:
made the changes, let me know if there's something else needed as well :)